tools/broccoli/broccoli-ts2dart.ts

```diff
@@ -4,17 +4,22 @@
 import fs = require('fs');
 import fse = require('fs-extra');
 import path = require('path');
-import * as ts2dart from 'ts2dart';
 import {wrapDiffingPlugin, DiffingBroccoliPlugin, DiffResult} from './diffing-broccoli-plugin';
 
 class TSToDartTranspiler implements DiffingBroccoliPlugin {
   static includeExtensions = ['.ts'];
 
-  private transpiler: ts2dart.Transpiler;
+  private transpiler: any /*ts2dart.Transpiler*/;
+  private additionalFiles: string[];
 
   constructor(public inputPath: string, public cachePath: string,
-              public options: ts2dart.TranspilerOptions) {
+              public options: any /*ts2dart.TranspilerOptions*/) {
+    this.additionalFiles = options.additionalFiles || [];
```

Comment:
Add a comment around what you _expect_ `additionalFiles` to be / what they're used for.